docs/examples/dns/auroradns/enable_disable_record.py

```diff
@@ -1,6 +1,5 @@
-from libcloud.dns.types import Provider
+from libcloud.dns.types import Provider, RecordType
```

Comment:
We tend to use one import per line - this makes searching and grepping a bit easier.